prow/plugins/project/project.go

```
@@ -0,0 +1,348 @@
+	}
+
+	// If proposedColumnName is not found (or not provided), add to one of the default
+	// columns. If none of the default columns exists, an error will be shown to the user
+	columnFound := false
+	proposedColumnID := 0
+	for _, c := range projectColumns {
+		if c.Name == proposedColumnName {
+			columnFound = true
+			proposedColumnID = c.ID
+			break
+		}
+	}
+	if !columnFound {
+		// If user does not provide a column name, look for the columns
+		// specified in the project config and see if any of them exists on the
+		// proposed project
+		if proposedColumnName == "" {
+			defaultColumn := ""
+			exists := false
+			if defaultColumn, exists = getColumnMap(projectConfig, org, repo)[proposedProject]; !exists {
+				// Try to find the proposedProject in the org config in case the
+				// project is on the org level
+				defaultColumn, exists = getOrgColumnMap(projectConfig, org, repo)[proposedProject]
+			}
+			if exists {
+				// See if the default column exists in the actual list of project columns
+				for _, pc := range projectColumns {
+					if pc.Name == defaultColumn {
+						proposedColumnID = pc.ID
+						proposedColumnName = pc.Name
+						columnFound = true
+						break
+					}
+				}
+			}
+		}
+		// In this case, user does not provide the column name in the command,
+		// or the provided column name cannot be found, and none of the default
+		// columns are available in the proposed project. An error will be
+		// shown to the user
+		if !columnFound {
+			msg = fmt.Sprintf(invalidColumn, proposedProject, projectColumns)
+			return gc.CreateComment(org, repo, e.Number, plugins.FormatResponseRaw(e.Body, e.HTMLURL, e.User.Login, msg))
+		}
+	}
+
+	// Move this issue/PR to the new column if there's already a project card for
+	// this issue/PR in this project
+	existingProjectCard, _ := gc.GetColumnProjectCard(proposedColumnID, e.Number)
```

Why are we ignoring this error?